### PR TITLE
Remove schema v1alpha2

### PR DESCRIPTION
**Pull request by gernest**
~This is a breaking change~

### CODE REVIEW

**Comment by gernest** [comment]
@thorfour have you considered tagging releases? Since we removed `v1alpha2` proto, loading `wal` files generated before this commit will fail.

**Review by thorfour — COMMENTED** [review]
We haven't discussed tagging versions yet. I could be convinced to tag it at v0.X.X to indicate none of the API is stable yet, but I don't think I'd like to go beyond that until we have less frequent breaking changes.

We should be able to do this change without breaking backwards compatibility, see comment below.

File: proto/frostdb/table/v1alpha1/config.proto [hunk 1]
```diff
@@ -3,20 +3,15 @@ syntax = "proto3";
...
-import "frostdb/schema/v1alpha2/schema.proto";
 
 // TableConfig is the configuration information for a table.
 message TableConfig {
-  // Schema of the table.
-  oneof schema {
-    // Deprecated schema definition. Used for backwards compatibility.
-    frostdb.schema.v1alpha1.Schema deprecated_schema = 1;
-    // Schema of the table. Use this field.
-    frostdb.schema.v1alpha2.Schema schema_v2 = 2;
```
[the remark this answers is not in the document]
Comment (gernest) [reply proto/frostdb/table/v1alpha1/config.proto:15]: nice catch! done 351a3d4

**Comment by gernest** [comment]
@thorfour  I addressed your comment. PTAL.

File: proto/frostdb/table/v1alpha1/config.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (gernest) [reply proto/frostdb/table/v1alpha1/config.proto:15]: I wasn't sure if you just wanted to reuse field `1` and reserve field `2`  or just delete  field `2` and reserve it, I went with the later .

**Pull request closed without merging**